doc/build_instructions/windows_msvc.md

```diff
@@ -28,13 +28,17 @@ __NOTE:__ You need to manually make sure and doublecheck if the system you are b
    - With the "Download debug binaries (...)" option enabled.
    - If in doubt, run the installer again and choose "Modify".
    - You are going to need the 64-bit version of python if you are planning to build the 64-bit version of openage, and vice versa.
+ 
+    __Note:__ You need to install Python 3.7.x currently, because 3.8 deals with finding DLLs differently (see #1201) which would result in errors down the line
```

Comment:
#1201 doesn't link from documents i think, so we would need to use a link here `[#1201](https://github.com/SFTtech/openage/issues/1201)`